packages/i18n/test/unit/scripts/update-csv.js

```diff
@@ -0,0 +1,4 @@
+const { update } = require("../../../dist/test/unit/helper/update-test-data");
+update('csv')
```

Comment:
It looks very odd to have javascript files here that are referencing dist. Perhaps we can use the same approach as in vocabularies package? https://github.com/SAP/open-ux-tools/blob/feat/1i8n-support-02/packages/odata-vocabularies/package.json#L29

Reply:
let's make  scope, functionality and importance of this file. 

It is used to update test data for csv parser. Sure certainly we could use `ts-node`, but this mean adding another dependency. Looking at memory zie it take roughly `1,2 MB` and importance of this file. I tend not to add that module and keep file size of this module thin.

Reply:
This would be just a dev dependency and would not affect the size of the package when installed and we already have it as a dependency in the repository. Also then we don't need to build the test folder (which we do not normally do in other modules either).